docs/manual/replacements_and_deletions.md

```diff
@@ -40,14 +40,6 @@ The following conditions are currently eligible for replacement:
 * `MissingProcesses`: This indicates that a process is not reporting to the database.
 * `PodFailing`: This indicates that one of the containers is not ready.
 
-## Enforce Full Replication
-
-With this setting enabled (defaults to `true`) the operator will check if the cluster has the desired fault tolerance and is available.
```

Comment:
Instead of deleting this complete block I would like to have a note here that the operator only removes ProcessGroups when the cluster has the desired fault tolerance and is available.